editor/plugins/text_control_editor_plugin.cpp

```diff
@@ -205,42 +290,54 @@ void TextControlEditor::_font_style_selected(int p_id) {
 }
 
 void TextControlEditor::_set_font() {
-	if (!edited_control) {
+	if (edited_controls.is_empty()) {
 		return;
 	}
 
 	UndoRedo *ur = EditorNode::get_singleton()->get_undo_redo();
 	ur->create_action(TTR("Set Font"));
 
-	if (font_list->get_selected_id() == FONT_INFO_THEME_DEFAULT) {
-		// Remove font override.
-		ur->add_do_method(edited_control, "remove_theme_font_override", edited_font);
-	} else if (font_list->get_selected_id() == FONT_INFO_USER_CUSTOM) {
-		// Restore "custom_font".
-		ur->add_do_method(edited_control, "add_theme_font_override", edited_font, custom_font);
-	} else {
-		// Load new font resource using selected name and style.
-		String name = font_list->get_item_text(font_list->get_selected());
-		String style = font_style_list->get_item_text(font_style_list->get_selected());
-		if (style.is_empty()) {
-			style = "Default";
+	int count = edited_controls.size();
+	for (int i = 0; i < count; ++i) {
+		Control *edited_control = edited_controls[i];
+
+		String edited_font;
+		if (edited_control->is_class("RichTextLabel")) {
+			edited_font = "normal_font";
+		} else {
+			edited_font = "font";
 		}
 
-		if (fonts.has(name)) {
-			Ref<FontData> fd = ResourceLoader::load(fonts[name][style]);
-			if (fd.is_valid()) {
-				Ref<Font> font;
-				font.instantiate();
-				font->add_data(fd);
-				ur->add_do_method(edited_control, "add_theme_font_override", edited_font, font);
+		if (font_list->get_selected_id() == FONT_INFO_THEME_DEFAULT) {
+			// Remove font override.
+			ur->add_do_method(edited_control, "remove_theme_font_override", edited_font);
+		} else if (font_list->get_selected_id() == FONT_INFO_USER_CUSTOM) {
+			// Restore "custom_font".
+			ur->add_do_method(edited_control, "add_theme_font_override", edited_font, custom_font);
+		} else {
+			// Load new font resource using selected name and style.
+			String name = font_list->get_item_text(font_list->get_selected());
+			String style = font_style_list->get_item_text(font_style_list->get_selected());
+			if (style.is_empty()) {
+				style = "Default";
+			}
+
+			if (fonts.has(name)) {
+				Ref<FontData> fd = ResourceLoader::load(fonts[name][style]);
+				if (fd.is_valid()) {
+					Ref<Font> font;
+					font.instantiate();
+					font->add_data(fd);
+					ur->add_do_method(edited_control, "add_theme_font_override", edited_font, font);
```

Comment:
```suggestion
					ur->add_do_method(edited_control, SNAME("add_theme_font_override"), edited_font, font);
```
etc. for other string literals passed when `StringName` is expected.